src/InstagramClone.Api/Controllers/UsersController.cs

```diff
@@ -36,5 +36,29 @@ public IActionResult GetUser(Guid id)
             return Ok(user);
         }
 
+
+
+        [HttpPost("")]
+        public async Task<IActionResult> PostUsers([FromBody] UserCreateDTO user)
+        {
+
+            if (user == null)
+                return BadRequest();
+            User userToInsert = new User() 
+            {
+                Id = user.Id,
+                Email = user.Email,
+                Password = user.Password,
+                FirstName = user.FirstName,
+                LastName = user.LastName,
+                Avatar = user.Avatar,
+                CreatedDate = DateTime.Now,
+            };
+            this._context.Users.Add(userToInsert);
+            this._context.SaveChanges();
+
+            return Ok(userToInsert);
```

Comment:
When a resource is created, the correct HTTP error code to return is 201 and not 200. 

You should check the docs, I think there's an appropriate method for this called `Created`. Find it and update this response type.